dotnet-9.yaml

```diff
@@ -106,6 +106,8 @@ subpackages:
     dependencies:
       runtime:
         - dotnet-9-runtime
+        - libexpat1
+        - libbrotlidec1
```

Comment:
This seems correct. I remember `libbrotlidec1` as a dependency even in Ubuntu. Expat is something new, but the error seems to clearly indicate it is needed.

One question however: why is it a runtime dep here in `aspnet-9-runtime` and not `dotnet-9-runtime`?